RELEASE.md

```diff
@@ -27,20 +27,24 @@ work. Here's one example workflow (assuming of course you have rights to
 directly push to the upstream repo).
 
 Create a temporary clone of the repo, e.g. to `/tmp`. Then create the `tag` and
-push to origin. For example, to create a release called `v0.24.0`, you would run
-the following commands:
+push to origin. Since knative 1.0 the releases in this repo still are versioned in the `v0.x.y` format, and the tags corresponds to the knative tags `knative-v1.x.y`. This starts from `v0.27.0` == `knative-v1.0.0` and so on. For example, to create a release called `v0.27.0`, you would run the following commands:
 
 ```shell
 cd /tmp
 git clone git@github.com:vmware-tanzu/sources-for-knative.git
 cd sources-for-knative
 
 # checkout the branch created above
-git checkout release-0.24
+git checkout release-1.0
 
 # associate the tag with the branch
-git tag -a v0.24.0 -m "Release v0.24.0"
-git push origin v0.24.0
+git tag -a v0.27.0 -m "Release v0.27.0"
+# since knative version 1.0 we need to create an aditional tag that matches the
+# knative-v1.x.y tag format
+git tag -a knative-v1.0.0 -m "Release knative-v1.0.0"
+
+git push origin v0.27.0
+git push origin knative-v1.0.0
```

Comment:
PR #353 adds `knative-*` as push target for creating the release. Are we now triggering two release runs for the same commit by pushing two tags?

Reply:
Was a mistake from my part on that #353 PR, now it's good